tachyon/zk/plonk/circuit/selector_description.h

```diff
@@ -28,6 +29,19 @@ class TACHYON_EXPORT SelectorDescription {
   const std::vector<bool>& activations() const { return *activations_; }
   size_t max_degree() const { return max_degree_; }
 
+  // Checks if two |activations| vectors are orthogonal.
+  // In a mathematical sense, two vectors are orthogonal if their dot product is
+  // zero. For boolean vectors, this means there is no position where both have
+  // true values.
+  bool IsOrthogonal(const SelectorDescription& other) const {
```

Comment:
As i know, the inner product of two orthogonal vectors is zero. 
The name of this function doesn't completely match that concept.